core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/DataInstructionRepository.java

```diff
@@ -57,6 +55,9 @@
         TextSubstring textSubstring = new TextSubstring();
         dataInstructions.put(textSubstring.getKeyword(), textSubstring);
 
+        ListSize listSize = new ListSize(executionRuntime);
+        dataInstructions.put(listSize.getKeyword(),listSize);
```

Comment:
code formatting. you can auto-format the code using IntelliJ.
Normally the Sonarlint plugin will also complain about this.